grammar_test/expressions.txt

```diff
@@ -264,3 +283,18 @@ B = lambda: True
   (expression_statement (assignment
     (expression_list (identifier))
     (expression_list (lambda (true))))))
+
+=====================================
+Conditional if expressions
+=====================================
+
+if a
+if a else b
```

Comment:
I don't think these are valid python. Testing them in the Python 2.7 REPL like so:

```python
>>> a = 1
>>> b = 2
>>> if a
```

I get this:

```python
  File "<stdin>", line 1
    if a
       ^
SyntaxError: invalid syntax
```

Same with the `if/else` version. I think some valid examples would be:

```python
doSomething() if someCondition()
doSomething() if someCondition() else doSomethingElse()
```